From 26fdad6e3d744b1890cf612e07d969350d173f4e Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Tue, 13 Oct 2015 16:50:25 +0800
Subject: [PATCH 0565/5242] MGS-1132 ARM: Kconfig: increase
 FORCE_MAX_ZONEORDER for ARCH_MXC

commit  b3b00caeb27c0970b19d09132cfc0e1bc1b2028a from
https://source.codeaurora.org/external/imx/linux-imx.git

Need increase the FORCE_MAX_ZONEORDER to 14 for high resolution camera
(GPU 2D user case). The default value 11(4MB) is not enough now.

Signed-off-by: Jason Liu <r64343@freescale.com>
(cherry picked from commit fff642ffe868cb55f5caec0501e36fd28b6ece50)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/Kconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index b8e3b18..dbe9544 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1728,6 +1728,7 @@ config FORCE_MAX_ZONEORDER
 	int "Maximum zone order"
 	default "12" if SOC_AM33XX
 	default "9" if SA1111 || ARCH_EFM32
+	default "14" if ARCH_MXC
 	default "11"
 	help
 	  The kernel memory allocator divides physically contiguous memory
-- 
1.7.9.5

